presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -2517,9 +2517,9 @@ private RelationType analyzeView(Query query, QualifiedObjectName name, Optional
                         .setRemoteUserAddress(session.getRemoteUserAddress().orElse(null))
                         .setUserAgent(session.getUserAgent().orElse(null))
                         .setClientInfo(session.getClientInfo().orElse(null))
-                        .setStartTime(session.getStartTime())
-                        .build();
-
+                        .setStartTime(session.getStartTime());
+                session.getConnectorProperties().forEach((connectorId, properties) -> properties.forEach((key, val) -> viewSessionBuilder.setConnectionProperty(connectorId, key, val)));
```

Comment:
we don't usually use `val`. We could either use `(key, value)` (which is preferred) or `(k, v)`